readers-frontend/e2e-tests/index.spec.ts

```diff
@@ -163,7 +163,7 @@ test.describe("Featured Articles", () => {
         );
         await expect(article.locator(".image-link")).toHaveAttribute(
           "href",
-          `/2023/02/24/test-article-7`
+          /^(.*?)\/2023\/02\/24\/test-article-7/
```

Comment:
No cause the String.raw syntax is needed for es6 strings. If I used that on the constant string the IDE flags it up as a warning cause it can just be a constant regexp string

Reply:
Ohh I see what you mean now. Cool that makes sense